src/icebreakers_api.rs

```diff
@@ -106,6 +123,34 @@ impl IcebreakersAPI {
 
             info!("Successfully registered with Icebreakers API.");
 
+            // In case we get a URI without a protocol (http: or https: or ws: or wss:):
+            let fallback_proto = if self.base_url.starts_with("https:") {
+                "wss:"
+            } else {
+                "ws:"
+            };
+
+            let success_response = SuccessResponse {
+                load_balancers: Some(
+                    success_response
+                        .load_balancers
+                        .into_iter()
+                        .flatten()
+                        .map(|lb| {
+                            if lb.uri.starts_with("//") {
+                                LoadBalancerConfig {
+                                    uri: format!("{}{}", fallback_proto, lb.uri),
```

Comment:
Optional: It would be nice to log that we are using a fallback protocol to the user.

Reply:
Fixed in 96ab45bf4809b21a89409c6b7319927327de56c3.